app/build.gradle

```diff
@@ -2,11 +2,12 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
-	namespace 'de.markusfisch.android.binaryeye'
-	compileSdkVersion sdk_version
```

Comment:
Why did you remove this line?

Reply:
I had moved it into the defaultConfig block, reverting that change now.